Client/Frontend/Browser/ClipboardBarDisplayHandler.swift

```diff
@@ -0,0 +1,82 @@
+
+import Foundation
+import Shared
+
+public struct UserDefaultClipboardKey {
+    public static let KeyLastSavedURL = "KeylastSavedURL"
+}
+
+public struct ClipboardBarToastUX {
+    static let DismissAfter = 10.0
+}
+
+protocol ClipboardBarDisplayHandlerDelegate: class {
+    func shouldDisplayClipboardBar(absoluteString: String)
+}
+
+class ClipboardBarDisplayHandler {
+    weak var delegate: ClipboardBarDisplayHandlerDelegate?
+    private var sessionStarted = true
+    private var prefs: Prefs
+    private var lastDisplayedURL: String? {
+        if let value = UserDefaults.standard.object(forKey: UserDefaultClipboardKey.KeyLastSavedURL) as? String {
```

Comment:
I was referring to the fact that the URL would be saved on the UserDefault which can be retrieved from /Library/Preferences/xyz.plist even if the history was cleared. Maybe I'm being too paranoid haha :) but ok, I'll change it to the in memory solution. Thanks for the feedback 👍